macro/run_vert_ca_its.C

```diff
@@ -30,18 +30,18 @@
 #include "Field/MagneticField.h"
 
 #include "ITSBase/GeometryTGeo.h"
-#include "ITStracking/Event.h"
+#include "ITStracking/ROframe.h"
 #include "ITStracking/IOUtils.h"
-#include "ITStracking/Vertexer.h"
+#include "ITStracking/VertexerBase.h"
 #include "ITStracking/ClusterLines.h"
 #include "MathUtils/Utils.h"
 #include "SimulationDataFormat/MCCompLabel.h"
 #include "SimulationDataFormat/MCTruthContainer.h"
 
-using o2::ITS::CA::Cluster;
-using o2::ITS::CA::Line;
-using o2::ITS::CA::MathUtils::calculatePhiCoordinate;
-using o2::ITS::CA::MathUtils::calculateRCoordinate;
+using o2::ITS::Cluster;
+using o2::ITS::Line;
+using o2::ITS::MathUtils::calculatePhiCoordinate;
+using o2::ITS::MathUtils::calculateRCoordinate;
```

Comment:
@mconcas Are these "usings" actually used anywhere later on inside the macro ?
If not, we should probably remove them.
The same concerns a few "includes".  Several Fair*.h headers seem to be there "for historical reasons" only.
Can it all be revised and cleaned at bit, please ?

Reply:
Hi @iouribelikov , thanks for the revision, a commit on this PR is on its way.
OK, indeed I forgot those `using o2::*`, they were there since long time, for debug purposes.
Please notice also that this macro will be soon removed as the legacy vertexer. Every change will be eventually deleted.
Anyway I agree also on cleaning up the `Fair*` things. Honestly I don't really know where to start. This macro has been _ducktyped_ from other macros.

Do you have a good pointer on an up-to-date method to do the same thing?

Reply:
@mconcas Ciao Matteo,
There might be even a better idea.  Cannot we simply drop this whole run_vert_*.C macro ?
(The other one, the run_test_vert_*.C, looks similar, and it is most likely cleaner as well)
Cheers,
Yura.

Reply:
@iouribelikov Ciao Iura,
yes, that's cleaner. 
However it is using the refactorised Vertexer version, since it will eventually integrate the `AliGPUReconstriction` interface.
I will open a PR on the AliTPCCommon repository, as soon as I do some tests.
That is why I am purposing this transition phase with a coexisting legacy Vertexer that can be used also in the run_tracker macro. 
Next pull request, after the merge of my contribution to AliTPCCommon, will:
 - remove VertexerBase class (legacy vertexer) and its macro -1k lines (-ish)
 - implement the new `AliGPUReconstruction` interface and make it a default for every place vertexer is used.

What do you think? Also this PR is needed to be merged to test the AliTPCCommon modification I am going to purpose.

Thanks and Cheers,
Matteo

Reply:
@mconcas  So we are on a transition. 
It does not make much sense to clean the pieces of code that will anyway disappear pretty soon.
@sawenzel OK. We can merge this PR.